concent_api/core/tests/test_integration_get_task_result.py

```diff
@@ -999,3 +999,116 @@ def test_concent_requests_task_result_from_provider_and_requestor_receives_task_
         self.assertEqual(message_from_concent.file_transfer_token.operation, FileTransferToken.Operation.download)
 
         self._assert_client_count_is_equal(2)
+
+    def test_concent_requests_task_result_from_provider_when_compute_task_def_deadline_is_float(self):
+        """
+        Tests if on requestor ForceGetTaskResult message Concent will return ForceGetTaskResultUpload
+        if provider uploads correct result, but compute_task_def['deadline'] is given as float type.
+
+        Expected message exchange:
+        Requestor -> Concent:     ForceGetTaskResult
+        Concent   -> Requestor:   AckForceGetTaskResult
+        Concent   -> Provider:    ForceGetTaskResultUpload
+        """
+
+        # STEP 1: Requestor forces get task result via Concent.
+        # Concent accepts request if all conditions were met.
+        compute_task_def = self._get_deserialized_compute_task_def(
+            task_id='99',
+            subtask_id='8',
+            deadline="2017-12-01 11:00:00"
+        )
+        compute_task_def['deadline'] = 1512126000.999999
```

Comment:
why is it overwritten? it can be given as argument to `_get_deserialized_compute_task_def` above.

Reply:
Test deleted